.mci.yml

```diff
@@ -60,7 +62,7 @@ functions:
                 git clone -b master https://github.com/mongodb/mongo-c-driver.git
                 cd mongo-c-driver
                 rm -rf /data/tmp/c-driver-install
-                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled
+                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-extra-align
```

Comment:
This avoids misaligned address errors like the following: https://evergreen.mongodb.com/task/mongodb_cpp_driver_dev_ubuntu1410_debug_ubsan_compile_and_test_de649bf1acba78db9f744c99f36a3655b540a2eb_16_01_14_15_57_52

I can add a comment.